package.json

```diff
@@ -5,16 +5,10 @@
   "main": "index.js",
   "scripts": {
     "lint": "./node_modules/.bin/standard",
-    "local-run": "./node_modules/.bin/node-lambda run -f ./config/local.env -j ./sample/sample_event.json",
-    "deploy-dev": "./node_modules/.bin/node-lambda deploy -e development -f ./config/dev.env -S config/event_sources_dev.json --role arn:aws:iam::224280085904:role/lambda_basic_execution --profile nypl-sandbox",
-    "deploy-prod": "./node_modules/.bin/node-lambda deploy -e production -f ./config/prod.env -S config/event_sources_prod.json --role arn:aws:iam::946183545209:role/lambda-full-access --profile nypl-digital-dev",
-    "package": "./node_modules/.bin/node-lambda package",
-    "setup-node-lambda-env": "cp ./sample/.env.sample ./.env",
-    "setup-local-env": "mkdir -p config && cp ./sample/deployment.env.sample ./config/local.env",
-    "setup-dev-env": "mkdir -p config && cp ./sample/deployment.env.sample ./config/dev.env",
-    "setup-prod-env": "mkdir -p config && cp ./sample/deployment.env.sample ./config/prod.env",
-    "setup-dev-sources": "cp ./sample/sample_event_sources.json ./config/event_sources_dev.json",
-    "setup-prod-sources": "cp ./sample/sample_event_sources.json ./config/event_sources_prod.json",
+    "run-local": "./node_modules/.bin/node-lambda run -f ./config/local.env -j event.json",
+    "deploy-development": "./node_modules/.bin/node-lambda deploy -e development -f ./config/development.env -S config/event_sources_development.json --role arn:aws:iam::224280085904:role/lambda_basic_execution --profile nypl-sandbox",
+    "deploy-qa": "./node_modules/.bin/node-lambda deploy -e qa -f ./config/qa.env -S config/event_sources_qa.json --role arn:aws:iam::946183545209:role/lambda-full-access --profile nypl-digital-dev",
+    "deploy-production": "./node_modules/.bin/node-lambda deploy -e production -f ./config/production.env -S config/event_sources_production.json --role arn:aws:iam::946183545209:role/lambda-full-access --profile nypl-digital-dev",
```

Comment:
No VPC?

Reply:
No. I believe there's no reason to put a service that doesn't connect to a database inside a VPC.